layers/+distribution/spacemacs-base/packages.el

```diff
@@ -1467,6 +1468,15 @@ ARG non nil means that the editing style is `vim'."
       (spacemacs|diminish whitespace-mode " ⓦ" " w")
       (spacemacs|diminish global-whitespace-mode " Ⓦ" " W"))))
 
+(defun spacemacs-base/init-ws-butler ()
+  (use-package ws-butler
+    :defer t
+    :if (eq 'changed dotspacemacs-whitespace-cleanup)
+    :init
+    (ws-butler-global-mode 1)
```

Comment:
If you enable the mode in `:init` there's no point to make it deferred. Just put everything under `:config` and remove the deferred.